docs/how-to-contribute.md

```diff
@@ -0,0 +1,53 @@
+
+Please refer to this guide for information on how to contribute to our data science work.
+
+## **GitHub Workflow**
+
+1. Create an issue for every task you plan to work on.
+2. Add the issue to the *New* column of the corresponding project board.
+3. Let somebody from the team review the issue and refine it, until it's clear what should be done and what defines done.
+   a. Provide some sort of acceptance criteria.
+   b. Break the issue into smaller issues if it can't be done in one sprint.
+4. Move the issue to the *To Do* column.
+5. Once you start work on the issue move to the *In Progress* column and assign it to yourself.
+6. Create a PR for the issue and reference the issue in the PR description.
+7. Let somebody from the team review the PR - never merge your own PRs.
+
+## **Contributing via Pull Requests**
+
+Contributions to another data science project take the form of pull requests (PRs) against that project’s GitHub repository. Here are the steps for contributing using PRs:
+
+1. Create a fork of the repository you wish to contribute to. The repo you fork from is referred to as the “upstream”. 
+2. Clone your fork to your working environment (either local or on jupyterhub). 
+3. Add the upstream repository you’ve forked from to your working repo: `git remote add upstream <upstream repo URL>`.
+4. Create and checkout a new branch to work on the issue/feature you’d like to implement: `git checkout -b branch-name` will both create and switch over to a new branch.
+5. Complete your work on this branch.
+6. When the work is complete, push your changes to a feature branch on your remote fork. 
+7. Make a PR from your fork’s feature branch to the upstream main.
+8. Wait for someone to review your PR. They may suggest or request changes to your PR before approving it.
+9. Once your PR is approved and merged into the repository, you can delete your branch. 
+
+## **Reviewing Pull Requests**
+
+Code reviews are common practice in the software engineering field and have been for some time. They provide developers with a well defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, enforcing quality assurance of code and standards for the engineering process across both the code base and the team itself. As members of the AI CoE in the office of the CTO, we believe that having a well defined approach for how to do the same thing with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process that should be addressed. Below are the 10 principles we use in our group to review data science work.
+
+1. Understand the Context : Make sure you read the pull request description and any linked issues so you know the data scientist's goal. This will help you provide an appropriate level of feedback and focus on the meaningful changes.
```

Comment:
Can you bold each of the point(s) titles? i.e. make "Understand the Context" -> **Understand the Context** and similarly bold all the other point titles